tools/s2n-events/src/main.rs

```diff
@@ -65,9 +65,12 @@ impl EventInfo<'_> {
             crate_name: "s2n_quic",
             input_path: concat!(
                 env!("CARGO_MANIFEST_DIR"),
-                "/../s2n-quic-core/events/**/*.rs"
```

Comment:
Longer term we should have the program take these as arguments and then have the caller pass them

Reply:
I was thinking of turning s2n-events into a library that exposed `Output`/everything else that's needed to generate an events system that this program currently uses. Then s2n-quic and s2n-tls could each own a program like this with a dependency on s2n-events.

But maybe keeping it as a program that's shared between s2n-quic and s2n-tls would work too, and you'd just configure everything with arguments to it. It might be less flexible, like you couldn't define custom tracing subscribers and stuff. But maybe you could just define that stuff in the shared program and select between them with arguments 🤔

Reply:
Yeah either could work!